src/foam/nanos/cron/Schedulable.js

```diff
@@ -147,7 +147,10 @@ foam.CLASS({
       section: 'summary',
       createVisibility: 'HIDDEN',
       gridColumns: 4,
-      order: 1
+      order: 1,
+      factory: function() {
+        return `${this.objectToSchedule?.toSummary()} ${this.objectToSchedule?.toSummary() && this.nextScheduleDate ? '-' : ''} ${this.nextScheduledDate}`;
```

Comment:
nextScheduledDate would be undefined in the case of a partially completed Schedulable. This would come from a DepositSchedule in Cards where we have everything else in the schedule set up, without the start date. This would occur in the case that we set up a subscription but don't actually know the exact date of when our first charge would be. This is because we are waiting for the first actual charge to determine the start date and the next scheduled date from that point on.